observability-lib/grafana/panels.go

```diff
@@ -33,6 +33,28 @@ func newQuery(query Query) *prometheus.DataqueryBuilder {
 	return res
 }
 
+type LegendOptions struct {
+	Placement   common.LegendPlacement
```

Comment:
won't this be better?

```suggestion
	Placement   common.LegendPlacement `default:common.LegendDisplayModeList`
```